### ISSUE TITLE
Make installation under Windows smoother.

### ISSUE THREAD

**Issue by jettisonjoe** [google/openhtf#455]
There are some issues when installing on Windows, i.e. a regular `pip install openhtf` doesn't work. In particular:
  * Some of our dependencies have their own dependencies that can't be fully handled by pip.
  * The `cryptography` package is pip installable, but needs extra pip flags and crashes pip install for openhtf.
  * If there ultimately _are_ non-pip install pre-reqs, we should document them in README.md.

@agdeepak, can you add detail based on your recent attempts to install on Windows?

**Comment by agdeepak** [comment]
Agreed with Joe's comments. Some observations/issues encountered during installation on Windows (using Joe's fork to bypass Windows install check issues):

1. First dependency error [during cryptography package install] -> Setup script exited with error: Microsoft Visual C++ 9.0 is required. Get it from http://aka.ms/vcpython27 
2. Openssl error-> fatal error C1083: Cannot open include file: 'openssl/opensslv.h': No such file or directory
3. After resolving above issues, started the web frontend and it (python) was blocked by windows default firewall. Worked fine after enabling Allow Python access thru firewall. 
4. Looking for some good examples to get started :)
